ge05_team4/models/__init__.py

```diff
@@ -0,0 +1 @@
+from . import sales_order_extend
```

Comment:
Don't add extend to the file name, sale.order.py it's the correct form for standard